TauMLTools.code-workspace

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Looks like this file was accidentally committed, please remove it.

Reply:
I thought it would be nice having the project file in the repository so one doesn't need to re-create it each time. But if you prefer, I can remove it.

Reply:
Oh sorry for my ignorance, when commenting I thought that it's just some internally used vscode (?) file. What would be its use-case scenario?

Reply:
usually I edit code with VSCode using remote ssh plugin and it is more convenient to have a workspace file. But it is also not a big deal to re-create it for each new installation. So up to you.

Reply:
That's interesting because I also use vscode+ssh remote but I've never set up anything like this (so ssh tunnels worked out of the box for me). But anyhow it's not a problem to keep the file.

Reply:
Yes it works, but sometimes it is nice to have project-dependent settings. For example, one silly use-case can be: set in `TauMLTools.code-workspace`
```
	"settings": {
		"editor.tabSize": 2
	}
```
to unify tabulation size within TauMLTools (assuming everyone is using vscode)

Reply:
Nice, good to learn about that, thanks! I am personally not familiar with the syntax to fill it now so that we have a unified code style across the group. So let's keep this file as it is now and changes can be introduced in the future.